freqtrade/exchange/exchange.py

```diff
@@ -795,7 +795,7 @@ def stoploss_adjust(self, stop_loss: float, order: Dict) -> bool:
         Verify stop_loss against stoploss-order value (limit or price)
         Returns True if adjustment is necessary.
         """
-        raise OperationalException(f"stoploss is not implemented for {self.name}.")
+        return stop_loss > float(order['stopPrice'])
```

Comment:
is `stopPrice` now unified across all exchanges in ccxt?

What will ccxt fill here if for some reason, there was no stop-price in the exchange response? will it be 0.0, or None, or missing (if an exchange doesn't have this part implemented yet?)

So far, we kept this method in the specific classes as it wasn't unified, so even if the method itself is mostly identical - we have to explicitly double-check if it's really working for this exchange.

Reply:
I'll add a `createStopOrder` method to CCXT, so that we can check for it
This is the progress on `stopPrice` https://github.com/ccxt/ccxt/issues/12044

Reply:
This is no longer in exchange though, I just created one for gateio